NEWS.rst

```diff
@@ -37,6 +37,7 @@ possible, especially the following contributors:
 - Markus Piotrowski
 - Suyash Gupta
 - Vini Salazar (first contribution)
+- Pascal Schläpfer (first contribution)
```

Comment:
Also in `NEWS.rst` the contributors are sorted by their first name.

Reply:
ok, sorry, I'm new to this :)